lib/auth0verification.js

```diff
@@ -60,20 +60,23 @@ const getAuthPublicKey = function (jwksUrl, cache, kid) {
 // secures the application with auth0, by implementing checks against
 // incoming jwt's, with configuration of what routes to apply it to
 module.exports = function (app, config, logger, cache) {
-  if (config && config.secret) {
-    // decode the client secret if it is base64 encoded
-    var secret;
-    if (config.secret.match(BASE64_REGEX)) {
-      secret = new Buffer(config.secret, 'base64');
-    } else {
-      secret = config.secret;
-    }
+  if (config) {
 
-    // set up jwt validation, which will take into account excluded routes
-    var v1JwtCheck = expressJwt({
-      secret: secret,
-      audience: config.clientId
-    });
+    if (config.secret && !config.disableV1) {
+      // decode the client secret if it is base64 encoded
+      var secret;

```

Comment:
I think we should figure out if we still should/need to support node 4 (and plan on adding babel as a future task) or if we can drop node 4 support and not need babel. @dpurrington thoughts?